Gifski/Intents.swift

```diff
@@ -1,6 +1,247 @@
 import AppIntents
 import AVFoundation
 
+enum CropEntity: AppEntity, Codable {
+	case noCrop
+	/**
+	 The exact coordinates in the image using a *bottom-left* origin (the rest of the app uses top-left origin)
+	 */
+	case exact(x: Int, y: Int, width: Int, height: Int)
+	case aspectRatio(width: Int, height: Int)
+
+	var id: String {
```

Comment:
I think this type should maybe rather be a TransientAppEntity. I think you can use the `optionsProvider` in the `crop` `@Parameter` to give it a default list of aspect ratios.